_posts/2021-04-17-mapping-tutorial.md

```diff
@@ -0,0 +1,193 @@
+* `smoothness=excellent`
+* `width=3.5`
+* `traffic_sign=DE:237`
+
+### Radstreifen
+
+
+> **Vorsicht**! Das `right` in `cycleway:right` bezieht sich immer auf die Laufrichtung des OSM Weges.
+> Und hier bezieht sich `smoothness` auf die Fahrbahn und `cycleway:right:smoothness` auf den Radstreifen.
+
+<img alt="Radstreifen in mäßigem Zustand" src="/assets/images/cycleway:lane.jpg" width= "200" class="float right">
+
+Eine Straße mit Radstreifen, der parallel zur Fahrbahn verläuft, hat möglicherweise folgende Tags:
+* `highway=residential`
+* `surface=asphalt` (Falls gleich für Fahrbahn als auch Radstreifen)
+* `cycleway:right=lane`
+* `cycleway:right:lane=exclusive`
+* `cycleway:right:smoothness=good`
+* `cycleway:right:width=1.7`
+* `cycleway:right:traffic_sign=DE:237` (nur, wenn beschildert)
+* `smoothness=excellent` (bezieht sich auf die Fahrbahn)
+
+> Falls der Radstreifen einen anderen Untergrund hat, als die Fahrbahn, kann dies durch `cycleway:right:surface=*` angezeigt werden.
+
+### Getrennter Rad-& Fußweg
+
+<img alt="Getrennter, ausgeschilderter Radweg und Fußweg" src="/assets/images/separated-cycleway.jpg" width= "200" class="float left">
+
+Ein ausgewiesener, getrennter Rad- und Fußweg könnte folgende Tags enthalten:
+* `highway=path`
+* `bicycle=designated`
+* `foot=designated`
+* `traffic_sign=DE:241` (nur wenn beschildert)
+* `segregated=yes`
+* `surface=paving_stones`
+* `cycleway:surface=*` (nur, falls sich vom Rest des Weges unterscheiden würde)
+* `cycleway:smoothness=intermediate`
+* `cycleway:width=0.7`
+
+### Bordsteine
+
+Bordsteine können insbesondere für Schwerlast-Lastenräder eine erhebliche Barriere darstellen und sollten deshalb in OpenStreetMap eingetragen werden. Sie werden als Punkt auf Wegen getaggt. Für den Mapathon sind Bordsteine entscheidend, welche als Punkte auf Radwegen liegen. Poller als Linien oder Poller, die nicht auf Radwegen liegen werden von uns nicht weiter beachtet.
+
+In Baden-Württemberg existieren auch von behördlicher Seite keine flächendeckenden Informationen über Bordsteine.
+
+<div class="flex c">
+  <figure role="group">
+    <img alt="Raised kerb" src="/assets/images/kerb_raised.jpg" width="200">
+    <figcaption>
+      <code>raised</code>
+    </figcaption>
+  </figure>
+    <figure role="group">
+    <img alt="lowered kerb" src="/assets/images/kerb_lowered.jpg" width= "200">
+    <figcaption>
+      <code>lowered</code>
+    </figcaption>
+  </figure>
+  <figure role="group">
+    <img alt="Flushed kerb" src="/assets/images/kerb_flushed.jpg" width="200">
+    <figcaption>
+      <code>flushed</code>
+    </figcaption>
+  </figure>
+</div>
+
+* [`barrier=kerb`](https://wiki.openstreetmap.org/wiki/DE:Tag:barrier=kerb)
+* [`kerb`](https://wiki.openstreetmap.org/wiki/Key:kerb)`=raised`
+* `height=0.07` (in Metern)
+
+Und auf Straßen, bei denen der Radweg auf diesen Wegen gemappt ist:
+
+* `cycleway:left:barrier=kerb`
+* [`kerb`](https://wiki.openstreetmap.org/wiki/DE:kerb)`=lowered`
```

Comment:
Wie ist das hier in OSM üblich? So wie von dir beschrieben, oder müsste das so sein?

* `cycleway:left:barrier:kerb=lowered`
* `cycleway:left:barrier:kerb:height=0.04`

Reply:
Es gibt scheinbar weder noch. Ich hätte ohne `barrier` vorgeschlagen, da sonst zu lange.